Makefile

```
@@ -144,15 +144,18 @@ clean:
 	$(Q) rm -rf build/dist
 
 discovery-rpms-%: VERSION = $(or $(LIGHTOS_VERSION),$(DEFAULT_REL))
+discovery-rpms-%: RPMOUT_DIR = $(WORKSPACE_TOP)/discovery-client/build/dist/$*
 discovery-rpms-%: build/dist build/discovery-client
-	$(Q) rm -rf build/dist/*
-	$(Q) rm -rf ${RPMOUT_DIR}
+	$(Q) rm -rf $(RPMOUT_DIR)
 	$(Q) rpmbuild -bb --clean --define="version ${VERSION}" --define="_builddir `pwd`" --define="dist $(DISCOVERY_CLIENT_RELEASE).$*" --define "_rpmdir $(RPMOUT_DIR)" discovery-client.spec
 
 discovery-client-debs: VERSION = $(or $(LIGHTOS_VERSION),$(DEFAULT_REL))
```

checkmake (0.3.2)

[warning] Target "discovery-client-debs" should be declared PHONY.

(phonydeclared)